keadm/cmd/keadm/app/cmd/common/types.go

```diff
@@ -155,6 +155,66 @@ type DiagnoseObject struct {
 	Use  string
 }
 
+// BatchJoinOptions has the kubeedge batch process information filled by CLI
+type BatchProcessOptions struct {
+	ConfigFile string
+}
```

Comment:
```suggestion
// BatchProcessOptions has the kubeedge batch process information filled by CLI
type BatchProcessOptions struct {
	ConfigFile string
}
```

Reply:
Thanks for your suggestion, I have revised it.